### DISCUSSION TITLE
Entity extraction feature along with custom NER

### DISCUSSION THREAD

**Discussion by OnlinePage** [iparsw/custom-intent#7] [Ideas]
Hi I find this library really impressive and would suggest to include features like custom as well as labeled entity extraction .
That would make the library really more fantastic and increase utility.
Thanks!

**Comment by iparsw** [comment]
I don’t think i can add this functionality into the same model that are built with chatbot class but entity extraction is in my todo list, im trying to make a package that makes creating simple ai model for other programmers in other fields easier, if you have any other suggestions I will be happy to hear them

**Reply by OnlinePage** [reply]
oo cool waiting eagerly for it
